js/src/personal/kazi/Kazi.page.tsx

```diff
@@ -45,14 +56,30 @@ export function KaziPage() {
 
       <button
         className={styles.sunToMoonButton}
-        onClick={sunToMoon}
+        onClick={changeIcon}
       >
         {sun ?
           <FaSun
-            className={sunAnimate ? styles.animateIcon : ''}
+            className={
+              animate === 'exitSun' ? styles.animationExit
+              : animate === 'enterSun' ?
+                styles.animationEnter
```

Comment:
Is this what prettier outputs for nested ternaries like this?
Does `pnpm run test` pass?